packages/node_modules/@node-red/nodes/core/network/22-websocket.js

```diff
@@ -210,27 +210,35 @@ module.exports = function(RED) {
             startconn(); // start outbound connection
         }
 
-        node.on("close", function() {
+        node.on("close", function(done) {
             if (node.heartbeatInterval) {
                 clearInterval(node.heartbeatInterval);
             }
             if (node.isServer) {
                 delete listenerNodes[node.fullPath];
                 node.server.close();
                 node._inputNodes = [];
-                activeListenerNodes--;
```

Comment:
Should this line be deleted? It may be voluntary or a mistake when you removed the following old commented code.

Reply:
If I remember right, this was an artifact/unused variable which I tidied up.

Reply:
If you're going to remove this line, then you should also remove the other references to it in the file - otherwise we now have a variable that increments with each new node, but never decrements.

Reply:
Most odd, I will re-check code but I thought I had done that. 🤔

Reply:
Fixed in [8df53e4](https://github.com/node-red/node-red/pull/3531/commits/8df53e441d419ce26ca53d062a0ac5b599f91438)